op-node/metrics/metrics.go

```diff
@@ -10,6 +10,8 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/libp2p/go-libp2p-core/metrics"
```

Comment:
It may be helpful to rename this to `p2pmetrics` or something that is less easily confused with the local package name.